staging/src/k8s.io/pod-security-admission/policy/check_hostNamespaces.go

```diff
@@ -25,15 +25,18 @@ import (
 )
 
 /*
-Sharing the host namespaces must be disallowed.
+Sharing the host namespaces must be disallowed.  The only exception is the user namespace as it is optional for now.
```

Comment:
avoid mentioning an unstable feature here. if you want to avoid confusion, can change this to something like:

```suggestion
Sharing the host network, PID, and IPC namespaces must be disallowed.
```